guide/admin/forumdescription.md

```diff
@@ -0,0 +1,21 @@
+## User Handbook
+### Changing the Forum Description
+
+
+Changing the description of your forum instance requires you to be in the Administrator Control Panel, you can do this by clicking your username to the top right of your screen, and then clicking **Administration**. The example below will show you how this is done:
```

Comment:
I thought long and hard about this, when we link to these documents in the forums, mostly it will be single pages. I included that as a standard at the top of the pages because A) itś in sequence with what needs to be done (I.E. They need to be in the Admin CP) and B) It prevents threads in the forum on *How do I get to that panel?* I did think about putting a notice in the index, but then we would in turn have to link to that before passing them the thread (and hope they read it). By having it at the top of the pages, itś much more difficult to miss. That said, I am happy to re-think it if its really necessary.